chromium_src/components/policy/core/browser/DEPS

```diff
@@ -0,0 +1,6 @@
+from brave_chromium_utils import inline_file
+inline_file("//components/policy/core/browser/DEPS", globals(), locals())
+
+include_rules = [
+  "+components/grit/brave_components_strings.h",
```

Comment:
wondering if you have a way to not need this. Maybe moving the string somewhere else?